### DISCUSSION TITLE
Bulk add recipes to organizers

### DISCUSSION THREAD

**Discussion by matthewkdies** [mealie-recipes/mealie#4768] [Feature Request]
### First Check

- [X] I used the GitHub search to find a similar requests and didn't find it.
- [X] Checked the [tasks tagged](https://github.com/mealie-recipes/mealie/issues?q=is%3Aissue+is%3Aopen+label%3Atask+) issues and verified my feature is not covered

### Please provide a concise description of the problem that would be addressed by this feature.

I have been poor with assigning tags, categories, and tools to recipes as recipes amassed. In order to properly enable meal planner rules in my household, I need these organizers applied to all recipes. With my number of recipes, it's unrealistic to use only the existing UI to apply organizers to the recipes.

Please note: If there's an existing workflow and for this and I didn't find it in the documentation nor existing issues or tasks, please let me know!

### Please provide a concise description of the feature that would resolve your issue.

Rather than using the below UI per-recipe in order to add organizers to a single recipe, I think navigating to an organizer should allow you to select recipes and add them. This might include an "Add Recipes" button which allows you to search and select recipes.

<img width="200" alt="image" src="https://github.com/user-attachments/assets/09c09f94-8c37-46a4-8815-a3f8cd0f5870" />

Alternatively, a selection on the recipe's page with the ability to add to an organizer would be powerful as well.

Finally, an API-only solution might be easiest. I tried manually edit the `database.json` file that's provided when downloading a backup with a quick script (I know, not my best decision 😄) but it failed and I had to restore from a backup.

### Please consider and list out some caveats or tradeoffs made in your design decision

Of the three options I provided above, I think the per-organizer solution is the most user-friendly. First off, It doesn't require adding selection and bulk editing to the recipes page. While it does introduce a bulk edit (no matter what), restricting the operation to only one type of edit (e.g., adding a single category, adding a single tag) to the selected recipes.

While the user experience might not be ultimately the most friendly, a bulk operation like this should be one that happens once (or rarely), as per-recipe operations can (and should!) still occur in the recipe editing.

### Additional Information

- [ ] If this is accepted I'm willing to submit a PR to provide this feature
- [ ] If this is accepted I'm willing to help maintain this feature
- [ ] I'm willing to sponsor/pay a developer to do this work

**Comment by matthewkdies** [comment]
I'd like to add an addendum to the above that I'm *very* tentatively open to trying a PR on this issue. Major emphasis on "trying"! 😆 I'm entirely unfamiliar with the codebase and don't have much relevant web-dev experience, so I might need some hand-holding from admins if this is desired.

**Comment by michael-genson** [comment]
This can be done from the data management page:
![398025793-88b6e96e-a966-4b62-9c39-30b7f849f7ad](https://github.com/user-attachments/assets/9bab129b-1502-4aed-8354-b6c75f51b685)

https://demo.mealie.io/group/data/recipes

**Reply by matthewkdies** [reply]
Thank you for the tip! Happy holidays 🎄

**Reply by michael-genson** [reply]
Happy holidays!!

**Reply by ronnybugle** [reply]
That's handy, but is there a way to filter recipes in that list? So for example, if I want to change all items with ingredients "flour" and/or category of "bread" to have a new tag of "baked"?
